### PR TITLE
publish npm

### PR DESCRIPTION
**Pull request by mgresko**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/no-response.yaml [hunk 1]
```diff
@@ -1,18 +0,0 @@
-#
```
Comment (emilhdiaz) [inline .github/no-response.yaml:1]: @mgresko did you intent to remove this file?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fhir-works-on-aws-routing",
+  "name": "@abacus-insights/fhir-works-on-aws-routing",
```
Comment (emilhdiaz) [inline package.json:2]: Do we need to update the dependencies below to point to `@abacus-insights/fhir-works-on-aws-interface`?

File: .github/no-response.yaml (hunk 1 above)
Comment (mgresko) [reply .github/no-response.yaml:1]: Yes. Didn’t think we wanted to auto close merge requests right now.

File: package.json (hunk 2 above)
Comment (mgresko) [reply package.json:2]: How else does publish know where to push to?

File: .github/no-response.yaml (hunk 1 above)
Comment (emilhdiaz) [reply .github/no-response.yaml:1]: Makes sense

**Pull request merged**